### PR TITLE
Adding Logspout and Splunk

### PR DESCRIPTION
**Pull request by marybel**

### CODE REVIEW

File: src/services.sh [hunk 1]
```diff
@@ -116,4 +116,49 @@ horde::service::rabbitmq() {
...
+    local logs=$(pwd)
+	local port_cfg="1514:1514"
+
+	horde::delete_stopped splunk || return 1
+
+	horde::cfg_hostname "${hostname}" || return 1
+
+	docker run -d \
+		-p $port_cfg \
+		-p "8000:8000" \
```
Comment (benschw) [inline src/services.sh]: do we need to expose port 8000 publicly? it looks like it is also configured through fabio at http://splunk.horde

Comment (marybel) [reply src/services.sh]: Good catch

**Comment by marybel** [comment]
Container name is consider host in Splunk now.
+ Fixed 8000 port exposure

**Comment by benschw** [comment]
Awesome! Thanks

**Pull request merged**